scripts/scriptscratch.js

```diff
@@ -0,0 +1,98 @@
+// mealplan app
+const mealApp = {}
+
+//api key 
+mealApp.key = 'cacba147c38b4f0faff0ce178c6edd1f'
+
+
+// gets user information and sends it to the getMeals function
+mealApp.getUserInfo = (userDiet, intollerances, calories, time) => {
+    // listen for diet to be selected and assign that to userDiet
+    // $('.checkbox-pair').on('click',)
+
+    // listen for intollerances to be selected and assign that to inlosserances
```

Comment:
I see _intolerances_ spelled incorrectly through out.. Just wondering if that will throw us errors?

Reply:
will fix